server/routes/api/disease.py

```diff
@@ -32,3 +37,47 @@ def get_node(dcid):
                                    post=False,
                                    has_payload=False)
   return response
+
+
+@bp.route('/diseaseParent/<path:dcid>')
+def get_disease_parent(dcid):
+  """Returns parent node for given a disease node."""
+  # list to store parent node
+  list_parent = []
+  # disease of final parent
+  DISEASE_DCID = "bio/DOID_4"
```

Comment:
nit: usually we like to define constants at the top of the file outside the function & maybe call it PARENT_DISEASE_DCID or something about the fact that the dcid is the final parent

Reply:
Done!